scratchpad.R

```diff
@@ -3,4 +3,5 @@ library(lubridate)
 library(dplyr)
 library(gsplot)
 library(repgen)
-fiveyeargwsum(fromJSON(system.file('extdata','fiveyeargwsum','fiveyeargwsum-with-discrete-example.json', package = 'repgen')), 'html', "Author")
+vdiagram(fromJSON(system.file('extdata','vdiagram','vdiagram-example.json', package = 'repgen')), 'html', "Author")
```

Comment:
Yes/no, it was there as a helper for my eclipse IDE setup, I accidentally checked it in while ago and decided to keep it. "Accidentally" committed the update again.